lib/ret_web/endpoint.ex

```diff
@@ -14,8 +14,7 @@ defmodule RetWeb.Endpoint do
     gzip: false,
     # Due to cloudfront, we want to include max-age in responses	
     cache_control_for_etags: "public, max-age=31536000",
-    only:
-      ~w(assets robots.txt favicon.ico hub-preview.png avatar-selector.html smoke-avatar-selector.html),
```

Comment:
So how does this PR handle `assets`? I can't tell.

Reply:
the end-user URLs for assets don't change after this is pushed, they still point to the cloudfront CDN for `assets-prod.reticulum.io`. however, the origin of that cloudfront URL will change. currently, the origin is reticulum, and this line made it so that the JS/CSS/etc files from the packed-in client code got served out statically (and then would be CDN cached from there.) once we push this out, we will update the CDN origin to a (new) S3 bucket dedicated for the Hubs client assets. The new CI build and deploy process for the client simply manipulates the state in this bucket. Reticulum is still responsible for going to that bucket to fetch the proper root html files (for when the non-CDN, root URLs are hit like the index page and the hub page), but the remainder of the assets (the digested JS and CSS, etc) will go directly from S3 -> CDN.

Reply:
Got it, thanks for explaining.